frontend/src/components/AssignmentDialog/AssignmentDialog.vue

```diff
@@ -104,6 +144,9 @@
 
   const { assignment } = storeToRefs(useAssignmentStore());
   const { closeAndResetAssignmentModalOrDialog } = useAssignmentStore();
+  const { contact } = storeToRefs(useContactStore());
+  const { phoneNumber } = storeToRefs(usePhoneNumberStore());
+  const { address } = storeToRefs(useAddressStore());

```

Comment:
so you are storing all store values as refs just to get 1 flag from each of them? This shounds like an overkill

Reply:
Done: removed it